server/issue.go

```diff
@@ -317,7 +317,7 @@ func httpAPIGetSearchEpics(ji Instance, w http.ResponseWriter, r *http.Request)
 
 	wg.Wait()
 
-	var result []ReactSelectOption
+	result := []ReactSelectOption{}
```

Comment:
why? So you get an empty array and not a nil in the response?

Reply:
Yes, search APIs I've used have always returned an empty array when nothing matches your search, and it makes sense to me to do the same. Do you think it should return nil instead?

Reply:
No, just curious - other than that it seems like all test changes.